### DISCUSSION TITLE
Why are there no updates?

### DISCUSSION THREAD

**Discussion by BugreporterOddworld** [AliveTeam/alive_reversing#1608] [General]
I know this is kinda _**dumb**_ to ask but when is the next _**update**_? also yes its me _**mariogamer12**_ i lost my old account,
but yeah i just wanna know if this is a _**dumb**_ thing to ask than just don't reply but yeah thats it
### _**i love this i use a Asus That uses AMD things (I don't do computer stuff) and it makes it hard to play the old games**_

**Comment by mouzedrift** [comment]
we're working on a bigger update [on another branch/fork](https://github.com/mouzedrift/alive_reversing/commits/abi_break_refcount/). there are still quite a few things that need to be done before it can be merged into the master branch.

**Reply by Mariogamer12** [reply]
Hey just saying i lost no longer have that account now and have this one can i get the grid for the editor i made a reddit post
can you send me a link (if it still even is there) edit: not for the reddit post for the grid image
